octopusdeploy/util.go

```diff
@@ -112,3 +112,10 @@ func stringHashCode(s string) int {
 	}
 	return 0
 }
+
+func formatBoolForActionProperty(b bool) string {
+	if b {
+		return "True"
+	}
+	return "False"
+}
```

Comment:
The `strconv.FormatBool` gives `true` and `false` which doesn't work properly in Server as it expects `True` and `False`, so I created a new helper method